src/main.rs

```diff
@@ -83,9 +83,15 @@ fn main() {
     let options = {
         let mut options: Options = Docopt::new(USAGE)
             .and_then(|d| {
+                let mut args: Vec<_> = std::env::args().collect();
```

Comment:
Could you explain why this code is needed? You mentioned in the PR that it is needed when doing `cargo run`, is that meant to be done inside the `cargo-outdated` repository for self-checks? Sorry I'm just not remembering when this is needed!

Reply:
AFAICT the docopt parser requires the first argument (past exe) matches what's in the usage string, which isn't the case when using eg. cargo run, which I was trying to use to verify my change.

Reply:
I'm not totally sure this one should be in the MR just because `cargo run` is such a narrow use-case. Normally, when I test I just do `export PATH=~/code/cargo-outdated/target/debug/:$PATH` and test in a few template crates that I have around from past issues.

I'm happy to discuss an a new issue, just want to get your other changes rolled in :D

Reply:
Sure np, I'll revert!